src/main/java/seedu/address/commons/util/FileUtil.java

```diff
@@ -109,9 +111,9 @@ public static void openPdfFile(String filePath) throws IOException {
      * @param filePath relative or absolute file path to PDF file
      * @return whether the PDF file exists in the file path
      */
-    public static boolean checkPdfFilePath(String filePath) {
-        File pdfFile = new File(filePath);
-        return pdfFile.exists();
+    public static boolean checkValidPdfFilePath(FilePath filePath) {
+        String path = filePath.value;
+        File pdfFile = new File(path);
+        return pdfFile.exists() && path.contains(".pdf");
```

Comment:
maybe instead of path.contains(".pdf") we can make use of a validation regex like the other ones to check that it ends with .pdf using $ e.g. VALIDATION_REGEX = ".+\\.pdf$" where '.+' ensures something before .pdf, '\\.' represents the '.' character and $ means it should end with .pdf

Reply:
Ok added thanks for the feedback!